src/scss/common/careers.scss

```diff
@@ -1,82 +1,129 @@
-    background-color: black;
-    padding: 0;
-    min-height: 300px;
-    min-width: 500px;
+  &-section {
+    padding: 2rem 0;
+    width: 100%;
+    background-color: #ffdf80;
   }
 
-  .video-description-column {
-    background-color: white;
-    padding: 2rem;
-    align-content: center;
+  &-background-img {
+    background-image: url("../../../public/assets/CareerBackground.svg");
+    background-repeat: no-repeat;
+    background-position: left;
+    background-size: contain;
   }
-}
 
-.career-title {
-  background-color: black;
-  color: white;
-  margin-bottom: 0;
-  padding: 0.5rem 2rem;
-  border-radius: .25rem;
-  text-align: center;
-}
+  &-title {
+    color: #120540;
+    font-size: 40px;
+    font-family: "dm-sans-light", DMSans-Light, sans-serif;
+    padding: 20px 0;
+  }
+
+  &-subtitle {
+    color: #120540;
+    font-size: 24px;
+    font-family: "dm-sans-medium", DMSans-Medium, sans-serif;
+    padding: 20px 30px;
+  }
 
-.cs-journey {
-  display: flex;
-  margin: 2rem 0 0 0;
-  .left-column {
-    flex-grow: 1;
-    min-width: 322px;
-    max-width: calc(100% - 322px);
-    padding: 2rem;
+  &-text {
+    color: #1c1c1c;
+    font-size: 19px;
+    font-family: "roboto", Roboto, sans-serif;
+    padding: 10px 30px;
+  }
+
+  &-media-row {
+    display: flex;
+    justify-content: space-between;
+    gap: 100px;
+    flex-wrap: wrap;
+  }
+
+  &-media-row > .col,
+  &-text-row > .col {
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+    justify-content: center;
+    align-items: center;
+    flex: 1;
+    min-width: 300px;
+    margin: 10px;
+  }
+
+  &-media-row-column,
+  &-text-row-column {
+    flex: 1;
+    width: 100%;
+    border-radius: 0 0 5px 5px;
     background-color: white;
     align-content: center;
   }
 
-  .right-column {
-    flex-shrink: 0;
-    padding: 0;
+  &-media-row-image,
+  &-media-row-video {
+    width: 100%;
+    height: 300px;
+    object-fit: cover;
+    border-radius: 5px 5px 0 0;
   }
 
-  .right-column img {
+  &-media-row-video iframe {
     width: 100%;
     height: 100%;
+    border-radius: inherit;
   }
-}
 
-@media (max-width: 1200px) {
-  .cs-journey,
-  .career {
-    max-width: 500px;
-    flex-direction: column;
-    iframe {
-      min-height: 300px;
+  @media (max-width: 768px) {
+    &-media-row {
+      flex-direction: column;
+      gap: 0;
     }
-    .video-column {
-      min-height: 0;
-    }
-    .video-description-column {
-      padding: 1rem;
+
+    &-media-row > .col,
+    &-text-row > .col {
+      padding: 10px;
     }
-    .video-column,
-    .video-description-column,
-    .left-column,
-    .right-column {
-      width: 100%;
+
+    &-media-row-column,
+    &-text-row-column {
       max-width: 100%;
-      min-width: 0;
     }
   }
-  .cs-journey {
-    .left-column {
-      padding: 1.5rem;
+
+  &-link-column {
+    text-align: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &-link {
```

Comment:
A good example of why SCSS is still needed as a pair for bootstrap :) 

I just ask you review all of the display and layout relating in this file please!

Reply:
Removed unused styling from the file.